lib/dot/definitions.def

```diff
@@ -3,7 +3,7 @@
     var $lvl = it.level;
     var $dataLvl = it.dataLevel;
     var $schema = it.schema[$keyword];
-    var $schemaPath = it.schemaPath + '.' + $keyword;
```

Comment:
> I'm not very familiar with dotjs

It's a normal templating engine that allows JavaScript in it. Whatever is inside {{  }} is just code, so it is copied as is, while things in {{= }} get into the output.

It's a bit easier to manage template then it would be to manage string concatenation to generate code, that's it, but it's the same really.